azureadps-2.0-preview/AzureAD/Get-AzureADApplication.md

```diff
@@ -57,7 +57,7 @@ Output:
 
     ObjectId                             AppId                                DisplayName
     --------                             -----                                -----------  
-    ed192e92-84d4-4baf-997d-1e190a81f28e 36ee4c6c-0812-40a2-b820-b22ebd02bce3 MyNewApp
+    00001111-aaaa-2222-bbbb-3333cccc4444 36ee4c6c-0812-40a2-b820-b22ebd02bce3 MyNewApp
```

Comment:
AppId value not updated.

Reply:
DocuTune is catching AppId in L51 and sees the same value in L60, but associated with object Id. The notation here is odd, so DocuTune is a bit off here.